src/adhocracy_core/adhocracy_core/authentication/__init__.py

```diff
@@ -227,7 +224,7 @@ def effective_principals(self, request: IRequest) -> list:
         if cached_principals:
             return cached_principals
         if self.authenticated_userid(request) is None:
-            principals = [Everyone, Anonymous]
+            principals = [Everyone]
```

Comment:
we can delete this. super().effective_principals should return [Everyone] in this case.